src/plots/wheelPlot.ts

```diff
@@ -0,0 +1,233 @@
+      let attrToProjector = super._propertyProjectors();
+      let rAccessor = Plot._scaledAccessor(this.r());
+      let r2Accessor = Plot._scaledAccessor(this.r2());
+      let tAccessor = Plot._scaledAccessor(this.t());
+      let t2Accessor = Plot._scaledAccessor(this.t2());
+      attrToProjector["d"] = (datum: any, index: number, ds: Dataset) => {
+        let t = tAccessor(datum, index, ds);
+        let t2 = t2Accessor(datum, index, ds);
+        if (t2 < t) {
+          t2 += (Math.floor((t - t2) / 360) + 1) * 360;
+        }
+        return d3.svg.arc().innerRadius(rAccessor(datum, index, ds))
+                           .outerRadius(r2Accessor(datum, index, ds))
+                           .startAngle(Utils.Math.degreesToRadians(t))
+                           .endAngle(Utils.Math.degreesToRadians(t2))(datum, index);
+      };
+      return attrToProjector;
+    }
+
+    /**
+     * Gets the AccessorScaleBinding for t in degrees.
+     */
+    public t<T>(): AccessorScaleBinding<T, number>;
+    /**
+     * Sets t to a constant number or the result of an Accessor<number> in degrees.
+     *
+     * @param {number|Accessor<number>} t
+     * @returns {Wheel} The calling Wheel Plot.
+     */
+    public t(t: number | Accessor<number>): Plots.Wheel<R, T>;
+    /**
+     * Sets t to a scaled constant value or scaled result of an Accessor in degrees.
+     * The supplied Scale will also be used for t2().
+     * The provided Scale will account for the values when autoDomain()-ing.
+     *
+     * @param {T|Accessor<T>} t
+     * @param {QuantitativeScale<T>} scale
+     * @returns {Wheel} The calling Wheel Plot.
+     */
+    public t<T>(t: T | Accessor<T>, scale: QuantitativeScale<T>): Plots.Wheel<R, T>;
+    public t<T>(t?: number | Accessor<number> | T | Accessor<T>, scale?: QuantitativeScale<T>): any {
+      if (t == null) {
+        return this._propertyBindings.get(Wheel._T_KEY);
+      }
+
+      if (scale != null) {
+        if (!QuantitativeScale.prototype.isPrototypeOf(scale)) {
+          throw new Error("scale needs to inherit from Scale.QuantitativeScale");
+        }
+        scale.range([0, 360]);
+        scale.padProportion(0);
+      }
+
+      let t2Binding = this.t2();
+      let t2Accessor = t2Binding && t2Binding.accessor;
+      if (t2Accessor != null) {
+        this._bindProperty(Wheel._T2_KEY, t2Accessor, scale);
+      }
+
+      this._bindProperty(Wheel._T_KEY, t, scale);
+      this.render();
+      return this;
+    }
+
+    /**
+     * Gets the AccessorScaleBinding for t2 in degrees.
+     */
+    public t2<T>(): AccessorScaleBinding<T, number>;
+    /**
+     * Sets t2 to a constant number or the result of an Accessor<number> in degrees.
+     * If a Scale has been set for t, it will also be used to scale t2.
+     *
+     * @param {number|Accessor<number>} t2
+     * @returns {Wheel} The calling Wheel Plot.
+     */
+    public t2(t2: number | Accessor<number>): Plots.Wheel<R, T>;
```

Comment:
Wrong signature? Should be `number | Accessor<number> | T | Accessor<T>`, as below.